.release/ci.hcl

```diff
@@ -0,0 +1,216 @@
+schema = "1"
+
+project "vault" {
+  team = "vault"
+  slack {
+    notification_channel = "CRF6FFKEW" // #vault-releases
+  }
+  github {
+    organization = "hashicorp"
+    repository = "vault"
+    release_branches = [
```

Comment:
Who uses this?

Reply:
This is read by the crt orchestrator, so it's the thing that decides if any workflows get kicked off beyond the build.yml.  Our general practice has been to list *all* crt enabled branches in the main branch's ci.hcl, though the only value that is actually functional here is "main". It's just listing the others so that humans know what other branches are enabled, but the other branches being enabled is dependent on their own ci.hcl. I've rambled a bit but hopefully that clears it up

Reply:
Yes, thanks